src/overlays/effects/ovl_Effect_Ss_Sbn/z_eff_ss_sbn.c

```diff
@@ -1,30 +1,208 @@
 /*
  * File: z_eff_ss_sbn.c
  * Overlay: ovl_Effect_Ss_Sbn
- * Description:
+ * Description: Popped Deku Bubble
+ *
+ * Note: The gfx field is used to store the colPoly, not anything gfx related
+ * This is not a problem since gfx is already void*
```

Comment:
Need to remember to review these later.

Reply:
I do plan on doing a full review/clean up PR once all the effects are in. Still not sure on exact documentation yet. Though I'll add a todo to remind myself to take a look.